packages/client/src/tlog/verify/body.ts

```diff
@@ -104,6 +132,42 @@ function verifyHashedRekordTLogBody(
   }
 }
 
+// Compare the given dsse v0.0.1 tlog entry to the given DSSE envelope.
+function verifyDSSE001TLogBody(
+  tlogEntry: Extract<ProposedDSSEEntry, { apiVersion: '0.0.1' }>,
+  dsse: sigstore.Envelope
+): void {
+  // Collect all of the signatures from the DSSE envelope
+  // Turns them into base64-encoded strings for comparison
+  const dsseSigs = dsse.signatures.map((signature) =>
+    signature.sig.toString('base64')
+  );
+
+  // Collect all of the signatures from the tlog entry
+  // Remember that tlog signatures are double base64-encoded
+  const tlogSigs = tlogEntry.spec.signatures?.map((signature) =>
+    signature.signature ? enc.base64Decode(signature.signature) : ''
+  );
```

Comment:
Is this comment about the double base64 encoding still true for the DSSE type?

Reply:
do you have docs somewhere on how to target staging with the client? I had to make several code changes when I was doing that.

Reply:
There's a new CLI in the "packages/cli" directory which supports a flag for overriding the rekor URL. From the root of the project you can do:

```
./packages/cli/bin/dev attest --help
```

That should show you the various flags